cmake/shared/PACE_CodeAnalysis.cmake

```diff
@@ -2,10 +2,14 @@ add_custom_target(analyse
   COMMENT "Performing code analysis..."
   )
 
+# Handle CMAKE list to matlab cell array
+string(JOIN "','" IGNORE_STRING ${PACE_MLINT_IGNORE})
+string(CONCAT IGNORE_STRING "{'" IGNORE_STRING "'}")
```

Comment:
Is it possible to perform this final wrap in `{'` ... `'}` as part of the command string you're building below?

Reply:
You *can*, but it adds extra brackets making it less legible. It's down to choice, but I prefer
`'exclude',${IGNORE_STRING});exit`
to
`'exclude',{'${IGNORE_STRING}'};exit`